src/Sylius/Bundle/MediaBundle/Resources/config/form.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+
+ This file is part of the Sylius package.
+
+ (c) Paweł Jędrzejewski
+
+ For the full copyright and license information, please view the LICENSE
+ file that was distributed with this source code.
+
+-->
+
+<container xmlns="http://symfony.com/schema/dic/services"
+           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xsi:schemaLocation="http://symfony.com/schema/dic/services
+                               http://symfony.com/schema/dic/services/services-1.0.xsd">
+
+    <parameters>
+
+        <!-- Form Types -->
+        <parameter key="sylius.form.type.image.class">Sylius\Bundle\MediaBundle\Form\Type\ImageType</parameter>
+
+    </parameters>
+
+    <services>
+
+        <!-- Form Types -->
+        <service id="sylius.form.type.image" class="%sylius.form.type.image.class%">
+            <argument>%sylius.model.image.class%</argument>
+            <argument>%sylius.validation_group.image%</argument>
+            <argument type="service" id="doctrine_phpcr.odm.document_manager" />
```

Comment:
I think only [PHPCR is only supported](http://symfony.com/doc/master/cmf/bundles/media/introduction.html) in CMF MediaBundle. 

> The bundle is built to support several persistence layers. Only the Doctrine PHPCR-ODM is implemented in version 1.0.

Anyway is there a way to abstract out the phpcr? Since we're heavily depending on `MediaBundle\Doctrine\Phpcr\Image` of that bundle?